### PR TITLE
Project List View

### PR DESCRIPTION
**Pull request by ash3679**

### CODE REVIEW

File: frontend/src/views/ProjectView.tsx
Comment (nuudlman) [inline frontend/src/views/ProjectView.tsx:1]: This is a different PR

File: frontend/src/styles/ProjectView.css
Comment (nuudlman) [inline frontend/src/styles/ProjectView.css:1]: Also a different PR

File: frontend/src/components/ProjectList.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+import '../styles/ProjectList.css'
+
+
+
+
+const projectsExisting = [
+  {name: 'Demo One Project', description: 'This is where the project description goes', id: 223},
+  {name: 'Demo Two Project', description: 'Project Description', id: 1523},
+  {name: 'Demo Three Project', description: 'Project Description', id: 449}
+]
```
Comment (nuudlman) [inline frontend/src/components/ProjectList.tsx:12]: Call the API instead of adding placeholder data

**Review by nuudlman — CHANGES_REQUESTED** [review]
Remove all files from the PR except for `components/ProjectList.tsx` and `styles/ProjectList.css`

File: frontend/src/components/ProjectList.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import { useState, useEffect } from "react";
+import useAuth from "../hooks/Auth.tsx";
+import "../styles/ProjectList.css";
+
+interface Project {
+    projectid: string;
+    name: string;
+    description: string;
+}
```
Comment (DrewBritt) [inline frontend/src/components/ProjectList.tsx:9]: This should probably moved to another file to be used by other components (ViewProject for example)

**Review by DrewBritt — APPROVED** [review]

**Review by nuudlman — APPROVED** [review]
Yeah we'll need to refactor some things but this is good for now

**Pull request merged**